frontend/src/components/frontpage/index.tsx

```diff
@@ -148,39 +187,137 @@ class FrontPage extends React.Component<Props, State> {
         }
     };
 
+    handleRadioButton = (value: string) => {
+        console.log(value);
+        if (value == AudioType.RSS) {
+            this.setState({ audioFiles: false });
+            return;
+        }
+        if (value == AudioType.AUDIO_FILES) {
+            this.setState({ audioFiles: true });
+            return;
+        }
+    };
+
+    submitDisabled = () => {
+        const {
+            podcastType,
+            rssFeed,
+            selectedFiles,
+            selectedAudioFiles,
+            processing,
+            audioFiles,
+        } = this.state;
+
+        // Require podcast type
+        if (!podcastType) return true;
+
+        // Don't upload if processing
```

Comment:
This comment is incorrect. Processing literally means it should upload, but the button itself should no longer be possible. The code is correct though.